### PR TITLE
added file tag

### PR DESCRIPTION
**Pull request by furkan-kayar-fg**
## Link to JIRA ticket

Add a link to the JIRA ticket that the changes in this PR describe.

### *What is the problem we're trying to solve*

added file tag

### *What changes have we introduced*

added file tag

Include any links to commits that introduce significant additions of code if they help explain the process of coming to the solution. e.g Addition of test database setup, addition of first test and production class, removal of buggy code, etc.

#### _Checklist_

- [ ] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [ ] Added tests to cover all new production code
- [ ] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [ ] Checked all code for possible refactoring
- [ ] Code pipeline builds correctly

### *Follow up actions after merging PR*

Are there any next steps that need to addressed after merging this PR? Add them here.

### CODE REVIEW

File: FinanceChargesListener/Gateway/Services/AwsS3FileService.cs [hunk 1]
```diff
@@ -163,13 +163,15 @@ public async Task<FileLocationResponse> UploadPrintRoomFile(IFormFile formFile,
 
             using (var stream = formFile.OpenReadStream())
             {
+                var tagSet = new List<Tag> { new Tag { Key = "fileType", Value = "Print Room" } };
```
Comment (sandipchandra) [inline FinanceChargesListener/Gateway/Services/AwsS3FileService.cs:166]: Avoid use space in File Tag Value

File: FinanceChargesListener/UseCase/GeneratePropertyChargesFileUseCase.cs [hunk 2]
```diff
@@ -161,7 +161,7 @@ public async Task ProcessMessageAsync(EntityEventSns message, JsonSerializerOpti
 
             var csvFileStream = new MemoryStream(Encoding.UTF8.GetBytes(builder.ToString()));
 
-            var fileName = $"Property Charges{DateTime.Now:yyyyMMddHHmmssfff}.csv";
+            var fileName = $"Property Charges {DateTime.Now:yyyyMMddHHmmssfff}.csv";
```
Comment (sandipchandra) [inline FinanceChargesListener/UseCase/GeneratePropertyChargesFileUseCase.cs:164]: Avoid using spaces

**Review by sandipchandra — CHANGES_REQUESTED** [review]

File: FinanceChargesListener/Gateway/Services/AwsS3FileService.cs (hunk 1 above)
Comment (FurkanKayar55) [reply FinanceChargesListener/Gateway/Services/AwsS3FileService.cs]: Done

File: FinanceChargesListener/UseCase/GeneratePropertyChargesFileUseCase.cs (hunk 2 above)
Comment (FurkanKayar55) [reply FinanceChargesListener/UseCase/GeneratePropertyChargesFileUseCase.cs]: Done

**Review by sandipchandra — APPROVED** [review]

**Pull request merged**